hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -771,6 +812,17 @@ func (r *NodePoolReconciler) delete(ctx context.Context, nodePool *hyperv1.NodeP
 		}
 	}
 
+	// Delete any ConfigMap belonging to this NodePool i.e. TunedConfig ConfigMaps.
```

Comment:
if you use labels, you could do `r.DeleteAllOf` instead of having a separate list function